### PR TITLE
Add comments to markdown templates

### PR DESCRIPTION
**Pull request by jacobherrington**
### Summary

Added comments to the areas of text that will be deleted from templates anyway.

### Other Information

This is a practice I have seen in a handful of repos, and personally find it convenient.

### CODE REVIEW

**Review by albertoalmagro — COMMENTED** [review]
Hello @jacobherrington,

thanks for the PR. I certainly prefer the current version instead of this one because when you see the template in GitHub (click raw) you see all the information and you are able to click on the links for the guides. See:

Current version: 
image
https://github.com/rails/rails/blob/master/.github/issue_template.md

vs proposed:
image
https://github.com/rails/rails/blob/dc27efccdd4424c4af15b0d4338c3428bd77bb10/.github/issue_template.md

**Review by gmcgibbon — APPROVED** [review]
I've also seen this on a few repos, but I don't feel strongly about it. Sometimes people don't remove these messages from their PRs and issues, which I assume is what this tries to address by hiding comments in parsed markdown. I'll approve, but I want another opinion before merging. Thoughts, @rafaelfranca?

**Pull request merged**
